### PR TITLE
Improve mobile navigation

### PR DESCRIPTION
**Pull request by rgoj**
This PR implements a mobile nav menu and brings back the subnav on mobile. Previously the subnavigation was just removed after a certain breakpoint, making many pages inaccessible.

I didn't want to overcrowd the main navigation on mobile, so I only put in the main links into the mobile menu. The subnavigation is now displayed as a separate ribbon that is always visible. Any better ideas welcome, doesn't have the amount of polish I would like it to have, but should be very functional, I think.

To test, please look at the navigation menu on the home page... but also on the other subpages (e.g. http://127.0.0.1:8000/schools/) - not every page has the subnavigation. As long as the subnav links are short, the subnav looks good, I think, but with longer labels (such as on the linked pages)... it lacks some polish. What I'd like to do would be to add some javascript to detect if the subnav spans multiple lines and if it does, make it stack horizontally. But I think it's functional as it is and I need to do 2 more cards that have some unknowns - I'll see if I can't get back to improving this tomorrow. As mentioned above, any better ideas welcome :)

### CODE REVIEW

File: static/js/mobileMenu.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+mobile_nav_button = document.querySelector('.nav__mobile-button');
+mobile_nav_label = document.querySelector('.nav__mobile-label')
+
+// Toggle aria-expanded for menu elements and change label
+function toggleMenu(element, label) {
+    let aria_expanded = element.getAttribute('aria-expanded');
```
Comment (rgoj) [inline static/js/mobileMenu.js]: Snippet gallery FTW, thanks @arh30 !

File: static/less/styles.less [hunk 2]
```diff
@@ -236,27 +248,138 @@ h1 {
...
+        fill: white;
+    }
+
+    .nav__mobile-button[aria-expanded="true"] + .nav__links {
+        max-height: 20em; // e.g. https://css-tricks.com/using-css-transitions-auto-dimensions/
+        transition: max-height 0.5s ease;
+    }
+}
+
+// Animate hamburger menu icon into an X icon
```
Comment (arh30) [inline static/less/styles.less]: beautiful

File: static/js/mobileMenu.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+mobile_nav_button = document.querySelector('.nav__mobile-button');
```
Comment (arh30) [inline static/js/mobileMenu.js]: awesome to get the aria stuffs in :) small thing but I think we try to stick to `mobile_menu.js` for file names!

Comment (rgoj) [reply static/js/mobileMenu.js]: Ah, good catch, cheers, I'll change the file name!

**Comment by arh30** [comment]
Going to cheekily throw this favicon on here too 😉could go on this PR or another but i just took the first bit of their logo, should do the job

**Pull request merged**